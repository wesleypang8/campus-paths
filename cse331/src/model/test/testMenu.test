#test menu

m

# All of your test scripts should end with q to terminate the program.
q