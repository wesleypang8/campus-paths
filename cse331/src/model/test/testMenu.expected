Menu:
    r to find a route
    b to see a list of all buildings
    q to quit

Enter an option ('m' to see the menu): #test menu

Menu:
    r to find a route
    b to see a list of all buildings
    q to quit

Enter an option ('m' to see the menu): 
# All of your test scripts should end with q to terminate the program.
